test/e2e/framework/framework.go

```diff
@@ -884,3 +884,12 @@ func GetLogToFileFunc(file *os.File) func(format string, args ...interface{}) {
 		writer.Flush()
 	}
 }
+
+// GetRuntimeClassHandler returns configured runtime handler.
+func GetRuntimeClassHandler(preconfiguredRuntimeHandler string) string {
```

Comment:
1. move `PreconfiguredRuntimeHandler` to here, make it private (`preconfiguredRuntimeHandler`)
2. Rename this function to `PreconfiguredRuntimeClassHandler`
3. Don't pass an argument, just return either docker or the preconfigured handler.